Samples.Objc/Samples.ObjC/ViewControllers/CheckoutViewController/CheckoutViewController.m

```diff
@@ -0,0 +1,376 @@
+                                                                                                                    breakdown:[[PPCPurchaseUnitBreakdown alloc] initWithItemTotal:[[PPCUnitAmount alloc] initWithCurrencyCode:PPCCurrencyCodeUsd
+                                                                                                                                                                                                                        value:itemTotal]
+                                                                                                                                                                         shipping:nil
+                                                                                                                                                                         handling:nil
+                                                                                                                                                                         taxTotal:[[PPCUnitAmount alloc] initWithCurrencyCode:PPCCurrencyCodeUsd
+                                                                                                                                                                                                                        value:taxTotal]
+                                                                                                                                                                        insurance:nil
+                                                                                                                                                                 shippingDiscount:nil
+                                                                                                                                                                         discount:nil]]
+                                                              referenceId:nil
+                                                                    payee:nil
+                                                       paymentInstruction:nil
+                                                  purchaseUnitDescription:nil
+                                                                 customId:nil
+                                                                invoiceId:nil
+                                                           softDescriptor:nil
+                                                                    items:self.items
+                                                                 shipping:nil];
+  NSArray<PPCPurchaseUnit *> *purchaseUnits = @[purchaseUnit];
+  PPCOrderRequest *order = [[PPCOrderRequest alloc] initWithIntent:PPCOrderIntentAuthorize
+                                                     purchaseUnits:purchaseUnits
+                                                             payer:nil
+                                                applicationContext:nil];
```

Comment:
What do you think about breaking the definitions onto separate lines ? like declaring the `PPCPurchaseUnitAmount` , `PPCPurchaseUnitBreakdown` separately to make it clearer how everything is being created. what do you think?

Reply:
yea I like that a lot!!!!!!! I'll make this change!!!!!

just wish that ObjC looks as clean as Swift tho haha